HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc

```diff
@@ -26,10 +26,10 @@ TriggerJSONMonitoring::TriggerJSONMonitoring(const edm::ParameterSet& ps)
 
   triggerResultsToken_ = consumes<edm::TriggerResults>(triggerResults_);
 
-  if (ps.exists("L1Results")) level1Results_ = ps.getParameter<edm::InputTag>("L1Results");  //DS 
-  else                        level1Results_ = edm::InputTag("hltGtDigis");                  //DS
+  if (ps.exists("L1Results")) level1Results_ = ps.getParameter<edm::InputTag>("L1Results");   
```

Comment:
Putting the proper value into the fillDescriptions(), the ps.exists is not needed!